.github/workflows/release.yml

```diff
@@ -325,8 +325,8 @@ jobs:
       - name: Cleanup repository before making changelog PR
         run: rm -rf .lock generation pulp_ostree_client* *-client.tar pulp_ostree.tar todo web docs.tar
 
-      - name: Stage changelog for master branch
-        run: python .github/workflows/scripts/stage-changelog-for-master.py ${{ github.event.inputs.release }}
+      - name: Stage changelog for main branch
+        run: python .github/workflows/scripts/stage-changelog-for-main.py ${{ github.event.inputs.release }}
```

Comment:
I have opened an issue for this change here: https://github.com/pulp/plugin_template/issues/526.